index.js

```diff
@@ -0,0 +1,2 @@
+const nombreUsuario = prompt('Ingrese su nombre.')
+alert(`Hola ${nombreUsuario}`);
```

Comment:
En este caso me parece que seria mejor no usar interpolación de strings.

```suggestion
alert('Hola ' + nombreUsuario);
```